var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -274,37 +272,56 @@ def cmake_args(self):
     #
     # export SPACK_USER_CACHE_PATH=/tmp/spack
     # export SPACK_DISABLE_LOCAL_CONFIG=true
-    def test(self):
-        if not self.spec.satisfies("@develop") or not os.path.isdir(self.prefix.bin):
-            tty.info("Skipping: checks not installed in bin for v{0}".format(self.version))
-            return
-
-        tests = [
-            ["NlpMdsEx1.exe", "400", "100", "0", "-selfcheck"],
-            ["NlpMdsEx1.exe", "400", "100", "1", "-selfcheck"],
-            ["NlpMdsEx1.exe", "400", "100", "0", "-empty_sp_row", "-selfcheck"],
-        ]
 
-        if "+raja" in self.spec:
-            tests.extend(
-                [
-                    ["NlpMdsEx1Raja.exe", "400", "100", "0", "-selfcheck"],
-                    ["NlpMdsEx1Raja.exe", "400", "100", "1", "-selfcheck"],
-                    ["NlpMdsEx1Raja.exe", "400", "100", "0", "-empty_sp_row", "-selfcheck"],
-                ]
+    def test_N1pMDsEx1(self):
+        """Test N1pMDsEx1"""
+
+        if not self.spec.satisfies("@0.6.1") or not os.path.isdir(self.prefix.bin):
+            raise SkipTest(
+                f"Skipping: checks not installed in bin for v{self.version}."
```

Comment:
`Skipping: ` is redundant, please remove.

Reply:
@cameronrutherford Would it be sufficient just to simply check that the executable is installed?

Reply:
Perhaps a more direct message for these two exceptions is `f"{exe} was not installed for version {self.version}"`, where `exe` is the executable.

Reply:
I am pretty confident with the version range here, but checking for the exe also would work